jsdoc-workspace.json

```diff
@@ -31,9 +31,9 @@
 	"openGraph": {
 		"title": "UI5 Tooling - API Reference",
 		"type": "website",
-		"image": "https://sap.github.io/ui5-tooling/docs/images/UI5_logo_wide.png",
```

Comment:
With https://github.com/SAP/ui5-tooling/pull/726 those should work again without this modification